book/src/using-bitfields.md

```diff
@@ -0,0 +1,84 @@
+# Using the Bitfield Types Generated by Bindgen
+
+## Bitfield Strategy Overview
+
+As Rust does not support bitfields, Bindgen generates an opaque struct for each with the following characteristics
+* Immutable getter functions for each bitfield named ```<bitfield>```
+* Setter functions for each bitfield named ```set_<bitfield>```
+* A static constructor  ```new_bitfield_{1, 2, ...}``` with a parameter for each bitfield contained within the opaque physical field.
```

Comment:
Nitpick: s/for each bitfield/for each contiguous block of bitfields/

There is a constructor for each of those physical, opaque fields that we end up generating, which is roughly equivalent to sets of contiguous bitfields.

Reply:
Done